appconfig/rdbsub.csv

```diff
@@ -0,0 +1,3 @@
+tabname,filts,columns
```

Comment:
Do we want a filtered subscription here? (I don't think we do)

Reply:
This is gone now